pkg/pillar/cmd/volumemgr/initialvolumestatus.go

```diff
@@ -269,7 +269,7 @@ func gcResetObjectsLastUse(ctx *volumemgrContext, dirName string) {
 }
 
 func deleteFile(filelocation string) {
-	if err := os.Remove(filelocation); err != nil {
+	if err := os.RemoveAll(filelocation); err != nil {
```

Comment:
This couldn't hurt, so why not? But why is it necessary?

Reply:
Actually, we are iterating over ```persist/runx/pods/prepared``` directory at bootup for populating initial status of container instances. So, we need to delete stale/obsolete container directories from the persist.

Reply:
I was not aware this was called by anything else. In any case, as we said, it certainly cannot hurt!

Reply:
> I was not aware this was called by anything else.
For reference, https://github.com/lf-edge/eve/pull/970